bigbluebutton-html5/imports/ui/components/audio/audio-modal/component.jsx

```diff
@@ -246,13 +247,14 @@ class AudioModal extends Component {
 
     const {
       joinEchoTest,
+      isConnecting,
     } = this.props;
 
     const {
       disableActions,
     } = this.state;
 
-    if (disableActions) return;
+    if (disableActions && isConnecting) return;
```

Comment:
Are you sure this is not meant to be `||` ?

Reply:
Yes. After the error happens,  disableActions = true, which means the code will always return (therefore audio-modal won't respond to user's input: this is the current problem). 
The AND guarantees the code will not return after the error happens (since after the audio error , isConnecting = false), making the whole sentence "(disableActions && isConnecting)" **false**. 

Basically, each flag work in combination for blocking/allowing user's input for two states:
**disableActions** doens't allow users to change modal _after_ it started first time, but allows the first time
**isConnecting** doesn't allow users to change modal after it started (at first, second, third time, ...) but allows after an error happens.